frontend/src/i18n/locales/features/user/transfer-ownership/en-sg.ts

```diff
@@ -0,0 +1,17 @@
+import { TransferOwnership } from '.'
+
+export const enSG: TransferOwnership = {
+  header: 'Transfer all forms',
+  form: {
+    label: 'Transfer ownership of all forms',
+    description:
+      'Share your secret key with this user for them to access response data',
+    button: 'Transfer ownership',
+  },
+  confirmation: {
+    transferringTo: 'You are transferring all forms to',
+    loseAccessWarning:
+      '. You will be removed as a collaborator and lose access to the forms you previously owned.',
```

Comment:
Can move this fullstop to be out of translations? To avoid missing fullstop issues when this text is removed.